lib/elixir/lib/calendar.ex

```diff
@@ -265,6 +265,15 @@ defmodule Calendar do
   """
   @callback valid_time?(hour, minute, second, microsecond) :: boolean
 
+  @doc """
+  Implements inspect for times, dates, datetimes, and "naive" datetimes.
+
+  The implementation must define inspect for `Date.t()`, `DateTime.t()` and
+  `NaiveDateTime.t()` structs.
+  """
+  @callback inspect(Time.t() | Date.t() | DateTime.t() | MaiveDateTime.t(), Inspect.Opts.t()) ::
```

Comment:
```suggestion
  @callback inspect(Time.t() | Date.t() | DateTime.t() | NaiveDateTime.t(), Inspect.Opts.t()) ::
```

Reply:
Arrrggggh. Thanks for the catch.  Getting very late this end and my eyes are beat.